docs/en/04_Changelogs/4.0.0.md

```diff
@@ -26,6 +28,8 @@
  * `ShortcodeHandler` interface to help generate standard handlers for HTML shortcodes in the editor.
  * `AssetNameGenerator` interface, including a `DefaultAssetNameGenerator` implementation, which is used to generate
    renaming suggestions based on an original given filename in order to resolve file duplication issues.
+ * `GeneratedAssetHandler` API now used to store and manage generated files (such as those used for error page
```

Comment:
Needs to mention all the changed method names for `Requirements`, e.g. `clear_combined_files()` -> `clearCombinedFiles()`